lldb/tools/driver/Driver.cpp

```diff
@@ -794,6 +800,10 @@ int main(int argc, char const *argv[]) {
   signal(SIGTSTP, sigtstp_handler);
 #endif
 
+  // Run the signal handling MainLoop on a separate thread.
+  std::thread signal_thread([] { g_signal_loop.Run(); });
+  signal_thread.detach();
```

Comment:
I'd like to avoid a detached thread running around  causing havoc during process destruction. Why not just join the thread right after the RequestTermination call?